### PR TITLE
Interfaces and helper classes

### PR DESCRIPTION
**Pull request by jdahmubed**

### CODE REVIEW

File: src/helper_classes/HttpArguments.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
+package helper_classes;
```
Comment (manu-sinha) [inline src/helper_classes/HttpArguments.java]: Please follow Jave style guide for naming convention. Refer https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names

File: src/helper_classes/HttpHeader.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
+package helper_classes;
```
Comment (manu-sinha) [inline src/helper_classes/HttpHeader.java]: I am checking with Lee about package name for our library. In any case please follow the style guide. Refer https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names

File: src/interfaces/Service.java [hunk 3]
```diff
@@ -0,0 +1,97 @@
+package interfaces;
```
Comment (manu-sinha) [inline src/interfaces/Service.java]: We shouldn't keep all the interfaces in one package. Interfaces should belong to their respective package that makes sense.
I am waiting for Lee to confirm the base package name and then we can decide package names.
@lccodes FYI

File: src/interfaces/State.java [hunk 4]
```diff
@@ -0,0 +1,104 @@
+package interfaces;
```
Comment (manu-sinha) [inline src/interfaces/State.java]: Like I mentioned above, we shouldn't keep all the interfaces together. Once I have the base package name, package name will need to change.

**Review by manu-sinha — CHANGES_REQUESTED** [review]
Hold on to changing the package names yet. I am waiting for Lee/Adam/Luke to let me know the base package name.

File: src/helper_classes/HttpArguments.java (hunk 1 above)
Comment (jdahmubed) [reply src/helper_classes/HttpArguments.java]: done

File: src/helper_classes/HttpHeader.java (hunk 2 above)
Comment (jdahmubed) [reply src/helper_classes/HttpHeader.java]: done

File: src/interfaces/Service.java (hunk 3 above)
Comment (jdahmubed) [reply src/interfaces/Service.java]: done

File: src/interfaces/State.java (hunk 4 above)
Comment (jdahmubed) [reply src/interfaces/State.java]: done

**Review by manu-sinha — APPROVED** [review]
LGTM

**Pull request merged**